galeracluster/source/training/tutorials/getting-started-kubernetes.rst

```diff
@@ -0,0 +1,275 @@
+.. meta::
+   :title: Getting Started Galera with Kubernetes
+   :description:
+   :language: en-US
+   :keywords:
+   :copyright: Codership Oy, 2014 - 2024. All Rights Reserved.
+
+
+.. container:: left-margin
+
+   .. container:: left-margin-top
+
+      :doc:`The Library <../../index>`
+
+   .. container:: left-margin-content
+
+      - :doc:`Documentation <../../documentation/index>`
+      - :doc:`Knowledge Base <../../kb/index>`
+      - :doc:`Training <../index>`
+
+      .. cssclass:: sub-links
+
+         - :doc:`Training Courses <../courses/index>`
+         - :doc:`Training Videos <../videos/index>`
+
+      .. cssclass:: sub-links
+
+         .. cssclass:: here
+
+         - :doc:`Tutorial Articles <./index>`
+
+      - :doc:`FAQ <../../faq>`
+      - :ref:`search`
+
+.. container:: top-links
+
+   - `Home <https://galeracluster.com>`_
+   - :doc:`Docs <../../documentation/index>`
+   - :doc:`KB <../../kb/index>`
+
+   .. cssclass:: here nav-wider
+
+      - :doc:`Training <../index>`
+
+   - :doc:`FAQ <../../faq>`
+
+
+.. cssclass:: library-article
+.. _`getting-started-kubernetes`:
+
+=======================================
+Getting Started Galera with Kubernetes
+=======================================
+
+.. rst-class:: article-stats
+
+   Length: 848 words; Published: October 22, 2024; Topic: Container; Level: Intermediate
+
+`Kubernetes <https://kubernetes.io/>`_ is an open source system for automating deployment, scaling, and management of containerized applications. In addition, `Helm Charts <https://helm.sh/>`_ help you define, install, and upgrade Kubernetes applications.
+
+In this tutorial, we will use the example of setting it up at `DigitalOcean <https://www.digitalocean.com/>`_, but the setup is similar, whether you do this in a Kubernetes on-premises deployment or if you do it on another cloud provider (such as `Amazon Elastic Kubernetes Service <https://aws.amazon.com/eks/>`_ or `Google Kubernetes Engine <https://cloud.google.com/kubernetes-engine/?hl=en>`_).
+
+Before you start, have a Linux computer to handle the management and connectivity to your Kubernetes cluster. In this example, we use Ubuntu 24.04 LTS. 
```

Comment:
Technically, you don't need a Linux box, but the tooling is easily available there (it is also on macOS)